frontend/src/containers/ThankYouPage/ThankYouPage.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './ThankYouPage.css';
+// import WelcomePage from '/containers/WelcomePage/WelcomePage';
+import { Button } from '@mui/material';
+import { amber } from '@mui/material/colors';
+
+const ThankYouPage: React.FC = () => {
+    return (
+      <>
+        <div className='content__thank-you'>
```

Comment:
Also, we are now transitioning to MUI. Please update the divs to Box or Containers. Thank you

Reply:
Use Typography component for p and h